dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/ClientIT.java

```diff
@@ -262,7 +262,7 @@ public void testDepsCommandWithUnknownRunners() throws IOException {
     @Test
     public void testDepsCommand() throws IOException {
         Client.main(new String[] { "--config", TestUtility.getConfigFileLocation(true), "deps" });
-        Assert.assertTrue(systemOutRule.getLog().contains("monotonic=="));
+        Assert.assertFalse(systemOutRule.getLog().contains("monotonic=="));
```

Comment:
It seems like this would pass if the log just had an empty string. 
If monotonic is no longer a dependency, this should test a different dependency

Reply:
Null is ommited. If it was false, it would outright fail some tests. These are all becoming hard refreshes because the default in the web service is true.

monotonic is the difference between python 3 and python 2.  Python 2 has it, python 3 doesn't. I've changed the default returned by this command to python 3 so it would not have it.

The test would not pass for a empty string because of the assertion function on the next line.

Reply:
> The test would not pass for a empty string because of the assertion function on the next line. 

Ah, thanks for the explanation. 

If asserting that monotonic is not installed is a way of ensuring that the environment is using Python 3 rather than Python2, add that intention as a code comment.